metadata.yaml

```diff
@@ -39,3 +39,10 @@ resources:
     type: oci-image
     description: Container image for Prometheus
     upstream-source: docker.io/ubuntu/prometheus@sha256:03936a740851b8786c328e08803a70feaa97731714a8dbe4b709c44366699b0c
+  promql-transform-amd64:
+    type: file
```

Comment:
I would add in the description where those binaries can be found, e.g., a link to the github release page

Reply:
Added a link to the project mainpage in the description. Thanks!